### DISCUSSION TITLE
To add a component, do you need to add relevant variables in packages\theme-chalk\src\common\var.scss

### DISCUSSION THREAD

**Discussion by lixiaofa** [element-plus/element-plus#12598] [Q&A]
To add a component, do you need to add relevant variables in packages\theme-chalk\src\common\var.scss
![image](https://user-images.githubusercontent.com/23230215/234159143-51e14b7b-1a70-42ed-b784-215f3eb76c5e.png)

What are the rules and what components can be added to them

**Comment by gimjin** [comment]
#11653 PR for the complete el-text component, hope it was helpful to you.

**Reply by lixiaofa** [reply]
> #11653 PR for the complete el-text component, hope it was helpful to you.

Actually, what I'm asking is this #12614

@gimjin
